CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [1.5.0] - 2022-11-02
+
+### Added
+- X.509-SVID and JWT-SVID TTLs can now be configured separately at both the entry-level and Server default level (#3445)
+- `k8s-workload-registrar` and `oidc-discovery-provider` CLIs now print their version when the `-version` flag is set (#3475)
+- Support for customizing SPIFFE ID paths of SPIRE Agents attested with the `azure_msi` NodeAttestor plugin (#3488)
+
+###  Changed
+- External plugins can no longer be named `join_token` to avoid conflicts with builtin plugin (#3469)
+- `spire-server run` command now performs TCP resolution of the configured bind address (#3421)
```

Comment:
Should be DNS resolution? Alternatively maybe we can just phrase this like so:
```suggestion
- `spire-server run` command now supports DNS names for the configured bind address (#3421)
```

Reply:
Oh ok, the PR description was inaccurate. Fixing.